widget/playground/src/containers/StyleLayout/StyleLayout.Preset.tsx

```diff
@@ -34,16 +36,24 @@ const EACH_COL_HEIGHT = 41;
 const TWO_ROWS_NUMBER_OF_COLS = 4;
 const ONE_ROW_NUMBER_OF_COLS = 2;
 export function Preset(props: PresetTypes) {
+  const { theme } = useConfigStore.use.config();
   const { tab } = props;
```

Comment:
When I choose one tab and then switch between tabs, my selected theme disappears. 
![image](https://github.com/rango-exchange/rango-client/assets/123297941/81b350b4-ab0e-4579-88d5-ee5d20188473)
![image](https://github.com/rango-exchange/rango-client/assets/123297941/485db7d3-4a54-48b8-a9c1-12ccdbdcd1e7)

Reply:
That was supposed to be it